tracer/test/test-applications/integrations/Samples.InstrumentedTests/Vulnerabilities/SqlInjection/DabaseHelpers/TestClasses.cs

```diff
@@ -5,4 +5,11 @@ public class Book
     public string Id { get; set; }
     public string Title { get; set; }
     public string Author { get; set; }
+
+    [DelegateDecompiler.Computed]
+    public string FullTitle
+    {
+        // [DelegateDecompiler.Computed]
```

Comment:
Can the attribute be applied here? If so I would add another test.

Reply:
I've added a test, but `DecompileDelegate` does not support it